specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json

```diff
@@ -435,15 +435,15 @@
               },
               {
                 "value": "OverQuota",
-                "description": "Reached daily cap quota, ingestion stopped."
+                "description": "Stopped ingestion following daily cap quota reach."
               },
               {
                 "value": "SubscriptionSuspended",
-                "description": "Ingestion stopped following suspended subscription."
+                "description": "Stopped ingestion following subscription getting into suspended state."
               },
               {
                 "value": "ApproachingQuota",
-                "description": "80% of daily cap quota reached."
+                "description": "80% of the daily cap quota reached."
```

Comment:
Remove cap. "80% of the daily quota reached."

Reply:
You don't want to say that as users can have payment considerations for daily cap.
"80% of the daily cap quota reached. Consider raising your daily cap limit and not loose any data"